pkg/pdf/pdf_test.go

```diff
@@ -3,14 +3,14 @@ package pdf_test
 import (
 	"bytes"
 	"fmt"
-	"github.com/johnfercher/maroto/internal"
-	"github.com/johnfercher/maroto/pkg/color"
+	"github.com/Vale-sail/maroto/internal"
```

Comment:
why to change import path? 
current repo would be deprecated?

Reply:
no, this is just to merge the utf feature